BlueprintUI/Sources/Internal/OSLog+Blueprint.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+import os.log
+
+extension OSLog {
```

Comment:
Nit: Could this go into the `Logger` file now?

Reply:
I wound up leaving it in a separate file along with the config to enable/disable.